huaweicloud/services/vpc/data_source_huaweicloud_networking_secgroups.go

```diff
@@ -72,7 +74,16 @@ func DataSourceNetworkingSecGroups() *schema.Resource {
 	}
 }
 
-func getSecGroupDetail(secGroup groups.SecurityGroup) map[string]interface{} {
+func flattenV1SecGroupDetail(secGroup v1Groups.SecurityGroup) map[string]interface{} {
```

Comment:
consider merge flattenV1SecGroupDetail and flattenV3SecGroupDetail into one func

Reply:
The current codes is clearer.
Not adopted

Reply:
done.